src/renderers/dom/client/ReactEventListener.js

```diff
@@ -126,10 +126,10 @@ function handleTopLevelWithPath(bookKeeping) {
       );
 
       // Jump to the root of this React render tree
-      while (currentPathElemenID !== newRootID) {
+      while (currentPathElementID !== newRootID) {
```

Comment:
Looks like something did sneak in. I'll clean it up and merge.

<img width="337" alt="screen shot 2015-07-09 at 11 48 31 pm" src="https://cloud.githubusercontent.com/assets/8445/8613552/1d446186-2695-11e5-8278-f008c86be8e3.png">